src/translation/english.c

```diff
@@ -585,10 +585,13 @@ static translation_string all_strings[] = {
     {TR_SIDEBAR_EXTRA_ENEMIES_DISTANT, "Enemies distant" },
     {TR_SIDEBAR_EXTRA_GODS, "Gods" },
     {TR_SIDEBAR_EXTRA_REQUESTS_VIEW_ALL, "View all" },
-    {TR_SIDEBAR_EXTRA_REQUESTS_HAVE, "have" },
     {TR_SIDEBAR_EXTRA_REQUESTS_SMALL_FORCE, "Small force"},
     {TR_SIDEBAR_EXTRA_REQUESTS_AVERAGE_FORCE, "Average force"},
-    {TR_SIDEBAR_EXTRA_REQUESTS_LARGE_FORCE, "Large force"}
+    {TR_SIDEBAR_EXTRA_REQUESTS_LARGE_FORCE, "Large force"},
+    {TR_SIDEBAR_EXTRA_REQUESTS_SEND, "Send request"},
+    {TR_SIDEBAR_EXTRA_REQUESTS_STOCK, "Store"},
+    {TR_SIDEBAR_EXTRA_REQUESTS_UNSTOCK, "Stop storing"},
+    {TR_SIDEBAR_EXTRA_REQUESTS_MONTHS, "Months"}
```

Comment:
You don't really need this string as base C3 already has one. Check below.

Reply:
I don't know how to use native string, where can I find the key correspondence ?